cms/navigation/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class NavigationConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```

Comment:
```suggestion
    default_auto_field = "django.db.models.AutoField"
```
I think that can be safely changed as we don't expect more than [2147483647](https://docs.djangoproject.com/en/5.1/ref/models/fields/#integerfield) Main Menus 😄.

In fact we probably don't expect to see that much of anything else so we could set it globally with [DEFAULT_AUTO_FIELD](https://docs.djangoproject.com/en/5.1/ref/settings/#default-auto-field) instead of setting it per app. Not related to this ticket, but a potential improvement in the future?